onadata/libs/utils/cache_tools.py

```diff
@@ -60,6 +60,9 @@
 # Cache timeouts used in XForm model
 XFORM_REGENERATE_INSTANCE_JSON_TASK_TTL = 24 * 60 * 60  # 24 hrs converted to seconds
 
+# Batch project IDs
+BATCH_PROJECT_IDS_CACHE = "batch_project_ids"
```

Comment:
IMO, the constant can be renamed to something like `PROJECT_DATE_MODIFIED_CACHE`